file-formats/dmon/README.md

```diff
@@ -0,0 +1,6 @@
+# DMON File Format
+This file format is used by the ADT editor for ABAP daemons (DMON).
```

Comment:
The purpose of the file formats is described in https://github.com/SAP/abap-file-formats#background-and-scope

We dont want to track usage in each definition, it will mess up the change history.

If SAP needs a list of where each file type is used, suggest moving that to a separate file

Reply:
Actually, we intended to make it more transparent for you and other people outside of SAP. If it is disturbing, we can just remove it. We don't need the information and have a separate list :)

Reply:
just wondering, if this object is supported in abapGit and other tooling, do we update this description? I think its a question users will ask, so we might want to store this somewhere

Reply:
We will remove this sentence from the readme